METIS/METIS_DET_IMG_N_GeoSnap.yaml

```diff
@@ -25,16 +27,23 @@ effects:
       kwargs:
           file_name: "!DET.qe_curve.file_name"
 
+    - name: auto_exposure
+      description: automatic determination of DIT and NDIT
+      class: AutoExposure
+      include: True
+      kwargs:
+        fill_frac: "!OBS.auto_exposure.fill_frac"
+        exptime: "!OBS.auto_exposure.exptime"
+
     - name: exposure_action
       description: Summing up sky signal for all DITs and NDITs
       class: SummedExposure
 
     - name: dark_current
       description: METIS GeoSnap dark current
       class: DarkCurrent
-      # [e-/s] level of dark current for each detector
       kwargs:
-          value: 100.
+          value: 1.e5   # [e-/s], E-REP-NOVA-MET-1191, v2-0
```

Comment:
`value : !!float 1.e5`

Unless there is new scientific notation detection in the newest pyyaml version. (Wouldn't suprise me if there is)

Reply:
Well spotted! Fixed.